backend/src/modules/doubles-tournament/core/domain/MatchesQueue.ts

```diff
@@ -28,7 +28,7 @@ export class MatchesQueue {
     return [...this.queue];
   }
 
-  withStartedMatch(match: QueuedMatch): MatchesQueue {
+  withStatusChangedMatch(match: QueuedMatch): MatchesQueue {
```

Comment:
Coś chyba ta metoda nie do końca mówi nam prawdę :) No bo dodaje nowy mecz do queue.

Reply:
enqueueMatchWithChangedStatus
?

Reply:
No, ale to nie jest changed status - tutaj cały mecz idzie. Nie widzę nic w implementacji co by zmieniało status. 
Co w sumie jak będą dwa mecze o tym samym matchNumber? Też się dodadzą.

Reply:
Nie będą, bo dodanie do kolejki jest wyżej - withEnqueuedMatch - i tam jest to pilnowane. Tutaj też jest to filtrowane, żeby wyciąć ten mecz o danym numerze i okej, zastąpić go nowym, ale budowanym na tych samych propertiesach za wyjątkiem statusu.

Reply:
A dobra, nie zwrocilem uwagi na to filtrowanie. Ale mi to wyglad na zamiane po prostu całego meczu, a nie tylko statusu. Tak to bym oczekiwał metody, ze masz podane id meczu i nowy status.